tripleblindmarket/private-kit
# Update styling and rename onboarding screens

#### Description:

Update the onboarding screens to match the latest Figma mockups, includes:
- Updating icons and background images
- Colors, spacing, and typography throughout
- Cleaning out and refactoring existing code
- Rename onboarding screens

#### Linked issues:

https://trello.com/c/7GRiaInh

#### Screenshots:

![Screen Shot 2020-06-30 at 12 01 10 PM](https://user-images.githubusercontent.com/2924479/86151207-6f341380-bacc-11ea-9be2-6e30332fa432.png)
![Screen Shot 2020-06-30 at 11 54 38 AM](https://user-images.githubusercontent.com/2924479/86151208-6fccaa00-bacc-11ea-9c63-df4b2d5d8628.png)
![Screen Shot 2020-06-30 at 11 54 28 AM](https://user-images.githubusercontent.com/2924479/86151209-6fccaa00-bacc-11ea-9e2c-a6dd41220ac0.png)
![Screen Shot 2020-06-30 at 11 54 19 AM](https://user-images.githubusercontent.com/2924479/86151210-70654080-bacc-11ea-8188-c325addf84a3.png)


#### How to test:

Pull the branch and view the screens locally to ensure they match the attached screenshots. These screenshots are referencing the Figma requirements here https://www.figma.com/file/79OwaKqHvSeriExpYzAvVN/PathCheck-Exploration-April-May-June-2020?node-id=6347%3A5337

Ensure all background images in both BT and GPS are still rendering
